instruqt-tracks/azure-cloud-account/config.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+containers:
+- name: cloud-client
+  image: gcr.io/instruqt/cloud-client
+  ports: [80]
+  shell: /bin/bash
+azure_subscriptions:
+- name: azuresubscription
+  roles:
+  - Contributor
```

Comment:
We should be consistent with the formatting of the files. I think Prettier should fix everything up, if you're not using it already. Specifically, I am referring to the "end of file newline" rule, which should be present.

We should add a code style guide to this repo 💡